### ISSUE TITLE
The new big video does not register views

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8771]
I have reported this before but it seems you don' believe this is an issue. Users are being robbed of views because if somebody views the big video on homepage it does not register views. If they was to go to the actual video in latest and play the video then the view registers. It does not register on the big version of it at the top of the homepage.

1. Go to the latest video on the site and look at its view count. https://ourtube.co.uk
2. Play the big video on the homepage for a minute.
3. Go back to the latest video and check its view count and you will see it doesn't change.

I was pointed to a page regarding view count by you and that is not the issue, big video has a bug.

**Comment by DanielnetoDotCom** [comment]
do you mean every time someone opens your home page should count as a view of the big video?

I am not sure if I agree with that. 

maybe if the person clicks in the video and opens it to listen its audio (MAYBE) it may count as a video view, I am not sure yet

**Comment by para2022** [comment]
No when playing it. Not opening the home page

**Comment by DanielnetoDotCom** [comment]
But the big video is autoplay, it will always play

**Comment by para2022** [comment]
Then I will have to remove it. Users are playing it rather than the original video on the video page. So the uploader gets no views because the big video doesn’t generate a view when playing it so it’s pointless.

**Comment by DanielnetoDotCom** [comment]
I see what do you mean, so the solution would be be when they click to expand the video then we will count a view . What do you think?

**Comment by para2022** [comment]
Yes that would be ideal

**Comment by para2022** [comment]
What would be really nice is maybe the same setup as the shorts on the big video. Il write some points below for the big video. Big video is a really nice addition but needs some additional work.

1. If the big video stays there all day and users click on that instead of going to the latest video page then that video gets no views whatsoever which is really bad for users. If it wasn't there they would go to the latest videos actual video page and play the video there which would then register views. Solution: Allow the big video to register a view when opened.
2. The big video shows live viewing count which is not true because its getting the live watching view count from users being on the main page causing confusion to the visitors and the uploader. The uploader thinks there is lots of activity on the big video when there isn't, they are just people active on the homepage. The uploader then goes to his video watch page and there are no views lol, based on the point raised above. I have had to disable the plugin as a work around. Solution: Disable 'live viewers' on the big video because It is actually not needed there really. 
3. There is no like/dislike function on the big video which is important. Their is also no video-logo-overlay or video owner. Solution: At the very least add like/dislike functions like on the shorts. Maybe take some inspiration from the shorts layout?

Hope this helps. Thanks

**Comment by DanielnetoDotCom** [comment]
This update will count views on big video when it is open

**Comment by para2022** [comment]
Thank you so much :)

**Comment by para2022** [comment]
doesnt work though dan you can check it on my site here https://ourtube.co.uk

**Comment by DanielnetoDotCom** [comment]
Looks like it is not updated properly,m can you make a git pull again?

**Comment by para2022** [comment]
git pull
Already up to date.

**Comment by DanielnetoDotCom** [comment]
I just sent something else to make sure you are up to date, please make a git pull again.

**Comment by para2022** [comment]
Still the same mate. Ive viewed his video on his video page already so it if they are IP based views it wouldnt register now anyway. As a work around i used my mobiles mobile IP but it still doesnt register a view. He has 2 views in his view info. One from me and one from him.

**Comment by para2022** [comment]
Shit hold on a sec lol

**Comment by para2022** [comment]
I did a git pull on the wrong server im sorry. ive done it on the right server now. I guess il have to wait for the next upload now mate. Im logged out on my mobile device and he now has 3 views one is an unknown user and i dont know if its me or someone else. Il keep track of things and report back. Thanks for your efforts though :)

**Comment by para2022** [comment]
His views are going up now so im asuming its working but like i say i will keep you posted. Thanks man!

**Comment by para2022** [comment]
Yup ive looked at what people are viewing in the websocket and i clicked what they are watching and some are watching that big video which correlates with his views going up. This is whats needed and may help with peoples views again now as they have been loosing a lot. Thank you so much Dan!!
